src/index.js

```diff
@@ -34,6 +34,26 @@ export function toggleCreator(context) {
   }
 }
 
+export function handleSumbit(func){
+  return (e) => {
+    e.preventDefault();
+
+    const form = e.target;
+    let values = {};
+
+    let children = [];
+
+    children.push(form.querySelectorAll('input, select, textarea'));
```

Comment:
most excellent news! lets use elements then (native apis are gettin good) woo

Ya don't really need to pull out the buttons. 
I'd do:
```
_.compact(
  elements
    .map(e => e.hasAttribute('name')))
    .map(e => ({ [e.name]: e.value }))
```
or something similar (I haven't run that, so it probably doesn't work. lol)